app/src/main/java/org/sopt/cgv/feature/time/component/TheaterSelectionModalFooter.kt

```diff
@@ -0,0 +1,150 @@
+package org.sopt.cgv.feature.time.component
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.ExperimentalLayoutApi
+import androidx.compose.foundation.layout.FlowRow
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Button
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.SheetState
+import androidx.compose.material3.Text
+import androidx.compose.material3.rememberModalBottomSheetState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.RectangleShape
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import kotlinx.coroutines.launch
+import org.sopt.cgv.core.common.extension.dropShadow
+import org.sopt.cgv.core.common.extension.noRippleClickable
+import org.sopt.cgv.core.designsystem.theme.White
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun TheaterSelectionModalFooter(
+    sheetState: SheetState,
+    onDismissRequest: () -> Unit,
+    selectedTheaters: MutableState<Set<String>>,
```

Comment:
여기서도
```suggestion
    selectedTheaters: Set<String>,
    onTheaterDeselected: (String) -> Unit
```
MutableState를 직접 전달하는 대신 불변 값과 이벤트 콜백을 전달하여
상태 변경 로직을 컴포넌트 외부로 이동시키고 칩 삭제 로직을 콜백 함수로 분리시키면 좋을 거 같아요! 이유는 같습니다!